src/main/resources/assets/bedrockify/lang/ru_ru.json

```diff
@@ -1,40 +1,154 @@
+  "bedrockify.loadingTips.70": "Вы можете разводить хоглинов с помощью багрового гриба, но это опасно!",
+  "bedrockify.loadingTips.71": "Используйте водоросли для привлечения и разведения морских черепах.",
+  "bedrockify.loadingTips.72": "Защитите маленьких черепах от враждебных мобов!",
+  "bedrockify.loadingTips.73": "Детские черепахи будут сбрасывать щитки, когда вырастут, их можно превратить в панцири черепахи.",
+  "bedrockify.loadingTips.74": "Приготовьте зелья для дыхания под водой для подводных!",
+  "bedrockify.loadingTips.75": "Дельфины могут привести вас к затонувшим кораблям и подводным руинам, накормить их треской или лососем и пойти по их следу!",
+  "bedrockify.loadingTips.76": "Используйте ножницы, чтобы вырезать тыкву.",
+  "bedrockify.loadingTips.77": "Вы знали, что даже можно поговорить с нашими разработчиками на discord.gg/Minecraft?",
+  "bedrockify.loadingTips.78": "Используйте фантомные мембраны, чтобы восстановить элитры.",
+  "bedrockify.loadingTips.79": "Морские соленья можно растопить в зеленовато-желтый краситель.",
+  "bedrockify.loadingTips.80": "Используйте сено для разведения лам!",
+  "bedrockify.loadingTips.81": "Компасы всегда указывают на мировое возрождение, создайте его, чтобы найти свой путь!",
+  "bedrockify.loadingTips.82": "Торгуйте с жителями, чтобы получить еду, инструменты и даже карты сокровищ!",
+  "bedrockify.loadingTips.83": "Вы знали, что можете держать карту в левой руке?",
+  "bedrockify.loadingTips.84": "Красться или носить ботинки ледохода, чтобы безопасно ходить по магматическим блокам.",
+  "bedrockify.loadingTips.85": "Собрать паутинку можно с помощью ножниц.",
+  "bedrockify.loadingTips.86": "Прежде чем добывать алмазы, красный камень или золото, убедитесь, что вы используете правильный тип кирки, иначе руда не подбрасывает.",
+  "bedrockify.loadingTips.87": "Золото больше в биомах Месы!",
+  "bedrockify.loadingTips.88": "Элитры можно использовать для планирования, найдите их на кораблях Край!",
+  "bedrockify.loadingTips.89": "Трезубцы выпадают из утопленников. Их можно держать и метать.",
+  "bedrockify.loadingTips.90": "В печке можно расплавить 20 элементов, используя блок сушеных водорослей как топливо.",
+  "bedrockify.loadingTips.91": "Наборы текстур меняют внешний вид блоков, предметов и меню в игре. Попробуйте их!",
+  "bedrockify.loadingTips.92": "Присоединяйтесь к обсуждению на discord.gg/Minecraft",
+  "bedrockify.loadingTips.93": "Изумрудные руды самые редкие в Minecraft! Они встречаются только на высокогорьях!",
+  "bedrockify.loadingTips.94": "Оцелоты задают трепку криперам!",
+  "bedrockify.loadingTips.95": "Диорит – его или любят, или ненавидят.",
+  "bedrockify.loadingTips.96": "Фантомы – это опасные мобы, которые являются по ночам. Не забывайте высыпаться!",
+  "bedrockify.loadingTips.97": "Зомби превращаются в утопленников, когда тонут.",
+  "bedrockify.loadingTips.98": "Зомби превращаются в утопленников, когда тонут.",
```

Comment:
This one is duplicate of 97 and does not correspond with the EN_US one.

Reply:
Oops! ill fix

Reply:
fixed